src/main/java/org/broadinstitute/hellbender/tools/copynumber/PreprocessIntervals.java

```diff
@@ -123,6 +145,24 @@
     )
     private File outputFile;
 
+    @Argument(
+            doc = "Intervals are layout in a grid so that bin start at fixed positions (grid points) 1 + n * bin_length. When intervals " +
+                    "do not exactly align with a grid starting position, the first bin in that interval will be smaller or simply skipped (see min-bin-length) " +
+                    "so that the rest of the  bins will start at a grid point",
+            fullName = GRID_LONG_NAME,
+            optional = true
+    )
+    private boolean grid = false;
+
+    @Argument(
+            doc = "The minimum bin length. Any truncated bin that are smaller than this length will be excluded, " +
+                    "when set to 1 means that no  bin will be excluded",
```

Comment:
excluded, when set to 1 means that no  bin will be excluded -> excluded.  When set to 1, no bin will be excluded.